### PR TITLE
Add soapysdr-module-airspyhf

### PR DESCRIPTION
**Pull request by ryanvolz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ryanvolz** [comment]
@conda-forge/staged-recipes Ready for review.

File: recipes/soapysdr-module-airspyhf/bld.bat [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    -DCMAKE_INSTALL_LIBDIR:STRING=lib ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+    ..
+if errorlevel 1 exit 1
+
+:: build
+cmake --build . --config Release -- -j%CPU_COUNT%
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/soapysdr-module-airspyhf/bld.bat:22]: Is there a `ctest` or `make check` we can run here?

Comment (ryanvolz) [reply recipes/soapysdr-module-airspyhf/bld.bat:22]: No, I don't see any tests in the source code. Our own functionality checks are about all that we can do, and even those are very simple.

File: recipes/soapysdr-module-airspyhf/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+mkdir build
+cd build
+
+cmake_config_args=(
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_INSTALL_LIBDIR=lib
+    -DCMAKE_INSTALL_PREFIX=$PREFIX
+)
+
+cmake ${CMAKE_ARGS} .. "${cmake_config_args[@]}"
```
Comment (xhochy) [inline recipes/soapysdr-module-airspyhf/build.sh:14]: No need for the long list here, only one of them is not part of `CMAKE_ARGS`
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release ..
```

**Pull request merged**